datashader/tests/benchmarks/test_quadmesh.py

```diff
@@ -34,8 +32,38 @@ def quadmesh_data(request, rng):
 def test_quadmesh_curvilinear(benchmark, quadmesh_data):
     def func():
         data, x_range, y_range = quadmesh_data
-        cvs = ds.Canvas(1024, 1024, x_range=x_range, y_range=y_range)
+        lon_coord, lat_coord = xr.broadcast(data.x, data.y)
+        data = data.assign_coords({"lon": lon_coord, "lat": lat_coord})
+        cvs = ds.Canvas(*CANVAS_SIZE, x_range=x_range, y_range=y_range)
         quadmesh = cvs.quadmesh(data.transpose("y", "x"), x="lon", y="lat")
```

Comment:
Is there a reason we're using transpose here? I just copied it over from one of your benchmark scripts.